### DISCUSSION TITLE
getComments having null for each comment thread

### DISCUSSION THREAD

**Discussion by EricRodabaugh** [LuanRT/YouTube.js#626] [Q&A] [answered]
I am using the getComments call and about 70% of the time when I try to use the information inside the comment thread, it's null. I have a proxy attached to the fetch implementation. This recently started happening when I updated to 9.1. Any idea on what might be happening?

**Comment by ChunkyProgrammer** [comment, marked as answer]
YouTube is testing a new format for comments. This should be fixed in 9.2.0

**Reply by EricRodabaugh** [reply]
Thanks! I updated one part of my system and I swore it didn't work, but when I tried again it seemed to work.
